specs/electra/beacon-chain.md

```diff
@@ -287,8 +287,8 @@ class AttesterSlashing(Container):
 class Attestation(Container):
     aggregation_bits: Bitlist[MAX_VALIDATORS_PER_COMMITTEE * MAX_COMMITTEES_PER_SLOT]  # [Modified in Electra:EIP7549]
     data: AttestationData
-    committee_bits: Bitvector[MAX_COMMITTEES_PER_SLOT]  # [New in Electra:EIP7549]
     signature: BLSSignature
+    committee_bits: Bitvector[MAX_COMMITTEES_PER_SLOT]  # [New in Electra:EIP7549]
```

Comment:
we do have the pattern that the signature in a given `Container` comes after the data it signs over (e.g. `SignedBeaconBlock` and `BeaconBlock`)

and this breaks this pattern which I don't love

Reply:
signature does not sign over committee_bits

Reply:
While that is historically true, as introduced in #625, it never got shipped that way.

The pattern that now exists is that either there is a wrapper containing `message` / `signature` fields, allowing `message` to be extended lateron. Or, it's like here where `signature` just signs 1 other field.

`committee_bits` and `aggregation_bits` are both not signed here. Only `data` is signed.